### PR TITLE
Update index.mdx

### PR DESCRIPTION
**Pull request by JennySanchez**
Taking the Wake card and link off

### Related Ticket(s)

{{Provide url(s) to the related ticket(s) that this pull request addresses}}

### Description

{{Add a human-readable description / detail summary of what the PR is changing and any details around how and why}}

{{If applicable, include a screenshot indicating an example or examples of what the PR is changing in the application}}

### Changelog

**New**

- {{new thing}}

**Changed**

- {{changed thing}}

**Removed**

- {{removed thing}}

### CODE REVIEW

**Review by annawen1 — APPROVED** [review]

File: src/pages/resources/index.mdx [hunk 1]
```diff
@@ -254,17 +254,6 @@ Carbon for IBM.com delivers the [IBM Design Language](https://www.ibm.com/design
...
-  
-  <Column colMd={4} colLg={4} noGutterSm>
-<ResourceCard
-  subTitle="Review design on Wake"
-  href="https://ibm.wake.com/space/37"
->
-
-![Sketch icon](../../images/wake_logo.svg)
-      
 </ResourceCard>
```
Comment (shixiedesign) [inline src/pages/resources/index.mdx:257]: Hey Jenny, you gotta remove this line too.

**Review by shixiedesign — CHANGES_REQUESTED** [review]

File: src/pages/resources/index.mdx (hunk 1 above)
Comment (annawen1) [reply src/pages/resources/index.mdx:257]: @shixiedesign It's fine, you can see above in on line 257 and 258 the
```
</ResourceCard>	
  </Column>
   ...
```
lines were removed, so we need to keep this `</ResourceCard></Column>`

It's the same if we didn't remove the first two and removed it on lines 268 & 270

Comment (shixiedesign) [reply src/pages/resources/index.mdx:257]: Thanks!

**Review by shixiedesign — APPROVED** [review]
Thank you! Sorry for the hold up :P

**Pull request merged**